assets/js/components/data.js

```diff
@@ -35,6 +36,27 @@ const {
 } = wp.hooks;
 const { __ } = wp.i18n;
 
+/**
+ * Sorts an object by its keys.
+ *
+ * The returned value will be a sorted copy of the input object.
+ * Any inner objects will also be sorted recursively.
+ *
+ * @param {object} data The data object to sort.
+ * @return {object} The sorted data object.
+ */
+const sortObjectProperties = ( data ) => {
```

Comment:
This should probably move to `assets/js/util`, even if this is the only place it's used, data.js is fairly big as it is.

> Disregard if this has already been done in a subsequent commit.